### PR TITLE
Group list route based on handler

### PR DESCRIPTION
**Pull request by agungsugiarto**
Each pull request should address a single issue and have a meaningful title.

- All bug fixes should be sent to the __"develop"__ branch, this is where the next bug fix version will be developed.
- PRs with any enhancement should be sent to the next minor version branch, e.g. __"4.3"__

**Description**
Group list route based on handler controller, output look like this:
image

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Commands/Utilities/Routes.php [hunk 1]
```diff
@@ -146,6 +146,11 @@ public function run(array $params)
             $tbody = [...$tbody, ...$autoRoutes];
         }
 
+        // Sort based on controller.
+        usort($tbody, function ($handler1, $handler2) {
+            return strcmp($handler1[2], $handler2[2]);
```
Comment (paulbalandan) [inline system/Commands/Utilities/Routes.php:151]: Do you really need case-sensitive sorting? Why not use `strcasecmp`?

Comment (agungsugiarto) [reply system/Commands/Utilities/Routes.php]: Already fixed, thank u @paulbalandan

**Comment by kenjis** [comment]
Thank you for sending a PR.

1. Why do you need this?
2. If this is not a bug fix, you should sent a PR to `4.3` branch

**Comment by kenjis** [comment]
I think this should be an option for the `spark routes` command.

**Comment by kenjis** [comment]
Closed by #7015

**Pull request closed without merging**